code/game/machinery/computer/message.dm

```diff
@@ -246,6 +266,37 @@
 				dat += {"<tr><td width = '5%'><center><A href='?src=\ref[src];deleter=\ref[rc]' style='color: rgb(255,0,0)'>X</a></center></td><td width='15%'>[rc.send_dpt]</td>
 				<td width='15%'>[rc.rec_dpt]</td><td width='300px'>[rc.message]</td><td width='15%'>[rc.stamp]</td><td width='15%'>[rc.id_auth]</td><td width='15%'>[rc.priority]</td></tr>"}
 			dat += "</table>"
+		if(5) //manual switchboard
+			var/a = 0
+			for(var/obj/machinery/requests_console/RC in requests_consoles)
+				var/shortname = RC.master_department_short + "-" + copytext(RC.department,1,switchboard_namelength-1)
```

Comment:
I like that you generate names instead of hardcoding them, I hate that you are completely fine with the fact that this generates five locations called u-Car, two named s-Sec, etc.

What I would do here is grab all capital letters from the name of the destination, so
Research and Development = n-RD
Research Director's Office = n-RDO
Cargo Bay = u-CB
Cargo Office = u-CO
Cargo Warehouse = u-CW
Cargo Dock = u-CD
Security Office = s-SO
Security Equipment = s-SE

etc etc

Reply:
>multiple locations with the same name

some consoles have the entire names identical - there's 5 RC's called "cargo desk" and they're even on the asteroid
i could give the duplicates a number at the end but i was hoping someone would code multiperson phonecalls so i can just dial all of them at once

capital letters are an excellent idea though

Reply:
on second thought, capital letters aren't that good(unless every RC gets renamed, but if we go that route might as well give them each a human-made shortname)
![image](https://github.com/vgstation-coders/vgstation13/assets/159179330/cd5cd405-9c06-4dc8-9319-76d02a16bb49)
aside from being much harder to decipher, strings apparently can't have multiple spaces in a row so it looks like shit